edx_rest_framework_extensions/tests/test_middleware.py

```diff
@@ -73,7 +73,12 @@ def test_request_standard_user_agent_metrics(self, mock_set_custom_metric):
     )
     @ddt.unpack
     @patch('edx_django_utils.monitoring.set_custom_metric')
-    def test_request_auth_type_token_metric(self, token, expected_token_type, mock_set_custom_metric):
+    def test_request_auth_type_token_metric(
+            self,
+            token,
+            expected_token_type,
+            mock_set_custom_metric,
+    ):
```

Comment:
Why split out this line?

Reply:
mistakenly thought it was over 120chars. same mistake below